docs/guide/docker.md

```diff
@@ -0,0 +1,114 @@
+---
+id: docker
+title: 🐳 Build docker image
+description: Build a simple docker image that can be used in production.
+sidebar_position: 8
+---
+
+After we have completed the development of our application, the moment comes when we want to upload it to the real world, most often we use Docker for this, wrapping our application in a container.
```

Comment:
Packaging instead of Wrapping. This paragraph could use some work.

Reply:
Sry, english is not my native language and I'm not sure what I get an idea. Can you give me an example?

Reply:
The paragraph is more suited for a blog or medium, not the documentation of a library